rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -128,7 +203,7 @@ error_code sys_ss_get_console_id(vm::ptr<u8> buf)
 {
 	sys_ss.notice("sys_ss_get_console_id(buf=*0x%x)", buf);
 
-	return sys_ss_appliance_info_manager(0x19003, buf);
+	return not_an_error(sys_ss_appliance_info_manager(0x19003, buf)); // not_an_error: Prevent duplicate error reports in emulator logs
```

Comment:
No duplicate error report, error_code::error_code(error_code&&) is a trivial constructor.

Reply:
Oh, I see. I didn't notice that.
Thank you very much!